domain/constants.go

```diff
@@ -0,0 +1,5 @@
+package domain
+
+const (
+	AuthenticatedEmailHeaderKey = "X-Goog-Authenticated-User-Email"
```

Comment:
if only appeal package uses it should we consider keeping it inside the package instead of domain?
it could be a normal private constant in appeal package. wdyt?

Reply:
yes currently it is only used in the appeal handler. This is actually more like a contract for application-wide, so I put it in the domain package instead. Also, there's a possibility that this constant will be used in other packages as well.
It is more proper to make this configurable actually, what do you think?

Reply:
usually my rule of thumb is if it is being shared across packages then domain would make sense.
If the possibility is pretty high then yeah it would probably okay to put it there.

But also another concern is that this constant is related to HTTP so imho keeping it as close to the handler using it would be better. wdyt?

Reply:
Alright, we'll see how to position this properly as this project grows. For now let's move it closer to the related use case, moving it to appeal handler for now.